incydr/cli/cmds/file_events.py

```diff
@@ -129,18 +132,39 @@ def search(
             risk_severity=risk_severity,
             risk_score=risk_score,
         )
-    events = client.session.post("/v2/file-events", json=query.dict())
-    events = events.json()["fileEvents"]
+
+    query.page_size = 10000
+
+    # skip pydantic modeling when output will just be json
+    if output or format_ in ("json", "raw-json"):
```

Comment:
I think I may have set the `table_format_option` to default to `None` while working on the logging implementation, but since we've scrapped anything other than raw-json for that we can probably put the `table` default value back.

Reply:
updated.